index.html

```diff
@@ -5,85 +5,99 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-social/5.0.0/bootstrap-social.css">
   <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css">
-  <link rel="stylesheet" href="/css/custom.css">
-</head>
-<body>
+
   <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-  <nav class="navbar navbar-fixed-top" id="header">
-    <div class="container">
+
+  <link rel="stylesheet" href="custom.css">
+
+  <div class="container">
+    <nav class="navbar navbar-fixed-center" id="header">
       <div class="navbar-header">
         <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
-            <span class="sr-only">Toggle Navigation</span>
-          </button>
+          <span class="sr-only">Toggle Navigation</span>
+        </button>
         <a class="navbar-brand secondaryColor" href="/">Farmer's Market International</a>
 
       </div>
 
-      <ul class="nav navbar-nav navbar-right">
-        <li><a href="#mission">Mission Statement</a></li>
-      </ul>
-      <ul class="nav navbar-nav navbar-right">
-        <li><a href="#solution">The Solution</a></li>
-      </ul>
-      <ul class="nav navbar-nav navbar-right">
-        <li><a href="#join">Sign Up!</a></li>
-      </ul>
-    </div>
-  </nav>
-  <div class="container" id="mission">
-  <div class="jumbotron">
+      <li class="nav navbar-nav navbar-right">
+        <ul><a href="#join">Join!</a></ul>
+      </li>
+      <li class="nav navbar-nav navbar-right">
+        <ul><a href="#solution">The Solution</a></ul>
+      </li>
+      <li class="nav navbar-nav navbar-right">
+        <ul><a href="#mission">Our Mission</a></ul>
+      </li>
+    </nav>
+  </div>
+
+
```

Comment:
dont put markup in the head. it only goes in the body. please either follow the directions i gave regarding the markup structure or review html tutorials online, this is one of the first things they say.